packages/error-reporting/.npmignore

```diff
@@ -0,0 +1,5 @@
+/bin
```

Comment:
With `.npmignore`, it can be easy to overlook excluding a new segment of files in a project. In the repo, we use the package.json `files` array to be explicit about what's included.